tests/Unit/Domain/Model/Subscription/SubscriberTest.php

```diff
@@ -247,6 +247,25 @@ public function setDisabledSetsDisabled()
         static::assertTrue($this->subject->isDisabled());
     }
 
+    /**
+     * @test
+     */
+    public function getExtraDataInitiallyReturnsEmptyString()
+    {
+        static::assertSame('', $this->subject->getExtraData());
+    }
+
+    /**
+     * @test
+     */
+    public function setExtraDataSetsExtraData()
```

Comment:
We also should have functional tests (for the repository) that test that this value can be read from the DB.

Reply:
thanks @oliverklee. I added it.